esphome/components/max31865/sensor.py

```diff
@@ -0,0 +1,26 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor, spi
+from esphome.const import CONF_ID, ICON_THERMOMETER, UNIT_CELSIUS
+
+DEPENDENCIES = ['spi']
+
+CONF_PIN_COUNT = 'even_pin_count'
```

Comment:
```suggestion
CONF_EVEN_PIN_COUNT = 'even_pin_count'
```

Use the same CONF_ name as the actual string.